tests/test-options.py

```diff
@@ -316,13 +316,13 @@ def test_debug_logger_object():
         assert dt.options.debug.enabled is True
 
         DT = dt.rbind([])
-        assert "dt.rbind([]) {" in logger.msg
+        assert "datatable.rbind([]) {" in logger.msg
         assert re.search(r"} # \d+(?:\.\d+)?(?:[eE][+-]?\d+)? s", logger.msg)
         logger.msg = ""
 
         with pytest.raises(TypeError):
             dt.rbind(4)
-        assert "dt.rbind(4) {" in logger.msg
+        assert "datatable.rbind(4) {" in logger.msg
```

Comment:
`dt` won't work for some reason?

Reply:
This is just a message that gets printed in the console. Previously the message said "dt.rbind" and now it's "datatable.rbind" because of the switch to XArgs.